resources/views/layouts/tables.blade.php

```diff
@@ -0,0 +1,68 @@
+@extends ('layouts.app')
+
+@yield ('variables')
+
+@section ('script')
+
+<script type='text/javascript' src='/DataTables/media/js/jquery.dataTables.min.js'></script>
+<script type='text/javascript' src='/DataTables/extensions/Buttons/js/dataTables.buttons.min.js'></script>
+<script type='text/javascript' src='/DataTables/extensions/Buttons/js/buttons.colVis.js'></script>
+
+<script type='text/javascript'>
+	var table;
+
+	$(document).ready(function(){
+	    table = $('#table').DataTable( {
+	        dom: 'lf<"columnVis"B>rtip',
+	        buttons: [
+	            'colvis'
+	        ]
+	    } );
+
+	$('#table tbody').on('click', 'td.details-control', function(){
+		var tr = $(this).closest('tr');
+		var rowClass = tr.class;
+		var row = table.row(tr);
+		if (row.child.isShown()){
+			row.child.hide();
+			tr.removeClass('shown');
+		}
+		else {
```

Comment:
Do you prefer the else being on a new line, or could you change it to be on the same line?